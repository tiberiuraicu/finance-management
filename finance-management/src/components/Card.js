import React from "react";
import { View, StyleSheet } from "react-native";

const Card = (props) => {
  return (
    <View style={{ ...styles.card, ...props.style }}>
      {props.children}
    </View>
  );
};

const styles = StyleSheet.create({
  card: {
    padding: 10,
    width: "95%",
    alignSelf: "center",
    paddingTop: "2%",
    paddingLeft: "3%",
    paddingRight: "3%",
    marginBottom: "2%",
    backgroundColor: "#FFFFFF",
    borderRadius: 10,
    shadowColor: "#000",
    shadowOffset: {
      width: 0,
      height: 2,
    },
    shadowOpacity: 0.23,
    shadowRadius: 2.62,
    elevation: 4,
  },
});

export default Card;
