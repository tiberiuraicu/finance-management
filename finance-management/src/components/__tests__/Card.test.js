import Card from "../Card";
import { shallow } from 'enzyme';

describe('Card component tests', () => {
    it('renders the component', () => {
      const wrapper = shallow(<Card />);
      expect(wrapper).toMatchSnapshot();
    });
  });