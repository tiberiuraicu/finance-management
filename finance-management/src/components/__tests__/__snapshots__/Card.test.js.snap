// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component tests renders the component 1`] = `ShallowWrapper {}`;
